src/openforms/forms/models/form.py

```diff
@@ -380,12 +397,27 @@ def __str__(self):
     @property
     def is_available(self) -> bool:
         """
-        Soft deleted, deactivated or forms in maintenance mode are not available.
+        Soft deleted, deactivated, forms in maintenance mode or forms which have reached the
+        submission limit are not available.
         """
-        if any((self._is_deleted, not self.active, self.maintenance_mode)):
+        if any(
+            (
+                self._is_deleted,
+                not self.active,
+                self.maintenance_mode,
+                self.has_reached_submissions_limit(),
+            )
+        ):
             return False
         return True
 
+    def has_reached_submissions_limit(self) -> bool:
```

Comment:
should be a `@property` instead of a method, it's a very simple calculated property.